extensions/registration-service/src/test/java/org/eclipse/dataspaceconnector/registration/store/InMemoryParticipantStoreTest.java

```diff
@@ -61,4 +61,14 @@ void listParticipantsWithStatus() {
         store.save(participant03);
         assertThat(store.listParticipantsWithStatus(AUTHORIZED)).containsOnly(participant01, participant03);
     }
+
+    @Test
+    void saveAndFindByDid() {
+        assertThat(store.findByDid(participant1.getDid())).isNull();
```

Comment:
it seems this should be two tests, (1) findById returns null and (2) save and findById

Reply:
Updated and also applied the same to listPartcipant test